src/Symfony/Component/Notifier/Bridge/Sinch/SinchTransportFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Sinch;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Iliya Miroslavov Iliev <i.miroslavov@gmail.com>
+ *
+ * @experimental in 5.0
```

Comment:
5.1

Reply:
Isn't this related to the notifier itself? It is experimental in 5.0 i'm not sure about how it would be in 5.1.

Reply:
good question 🤔 

friendly ping @nicolas-grekas

Reply:
We are going to keep it experimental in 5.1.

Reply:
The requested changes are applied.